WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -236,7 +236,7 @@ public static KeyManager FromFile(string filePath)
 
 		var km = JsonConvert.DeserializeObject<KeyManager>(jsonString);
 
-		km.SetFilePath(filePath);
+		km!.SetFilePath(filePath);
```

Comment:
Nop. The compiler is telling you that that variable can be `null` because **it can be null**. By muting the compiler we are losing a valuable friend.

Here you don't know whether `km` is `null` or not and the only way to be sure is by checking its value.

Try with something like this:

```c#
		if (km is null)
		{
			throw new FormatException($"Wallet file at: `{filePath}` is not a valid wallet file or it is corrupted.");
		}
```